index.html

```diff
@@ -605,12 +610,14 @@ <h2>Queue a <code>PerformanceEntry</code></h2>
         <li>For each <var>observer</var> in <var>interested observers</var>:
           <ol>
             <li>Append <var>new entry</var> to <var>observer</var>'s
-            <a>observer buffer</a>.
+            <a>observer buffer</a> for that <a data-lt=
```

Comment:
This seems wrong. A single PerformanceObserver still has only one buffer, which contains all of the entries, no?

Reply:
Right, done.